io.catenax.pcf/4.0.0/Pcf.ttl

```diff
@@ -0,0 +1,896 @@
+   samm:see <https://wbcsd.github.io/tr/2023/data-exchange-protocol-20230221/> ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "2.0.1-20230314" .
+
+:partialFullPcf a samm:Property ;
+   samm:preferredName "Partial Or Full PCF (Product Carbon Footprint)"@en ;
+   samm:description "Indicator for partial or full PCF (Product Carbon Footprint) declaration as specified in the Catena-X PCF Rulebook."@en ;
+   samm:characteristic :PartialFullPcfCharacteristic ;
+   samm:exampleValue "Cradle-to-gate" .
+
+:precedingPfIds a samm:Property ;
+   samm:preferredName "Previous PCF (Product Carbon Footprint) Identifiers"@en ;
+   samm:description "Set of preceding PCF (Product Carbon Footprint) identifiers without duplicates as specified in the Catena-X PCF Rulebook in accordance with the WBCSD (World Business Council for Sustainable Development) Pathfinder framework and the technical specifications for PCF Data Exchange (Version 2.0.0) from the WBCSD/ PACT initiative. Declared as \"optional\" in WBCSD, needs to be covered by application."@en ;
+   samm:see <https://wbcsd.github.io/tr/2023/data-exchange-protocol-20230221/> ;
+   samm:characteristic :PrecedingPfIdsTrait .
+
+:version a samm:Property ;
+   samm:preferredName "Product (Carbon) Footprint Version"@en ;
+   samm:description "Version of the product (carbon) footprint as specified in the Catena-X PCF Rulebook in accordance with the technical specifications for PCF Data Exchange (Version 2.0.0) from the WBCSD (World Business Council for Sustainable Development)/ PACT initiative. In Catena-X set to \"0\" per default."@en ;
+   samm:see <https://wbcsd.github.io/tr/2023/data-exchange-protocol-20230221/> ;
+   samm:characteristic :ProductFootprintVersion ;
+   samm:exampleValue "0"^^xsd:positiveInteger .
+
+:created a samm:Property ;
+   samm:preferredName "Creation of the Product (Carbon) Footprint"@en ;
+   samm:description "Timestamp of the creation of the Product (Carbon) Footprint as specified in the Catena-X PCF Rulebook in accordance with the technical specifications for PCF Data Exchange (Version 2.0.0) from the WBCSD (World Business Council for Sustainable Development)/ PACT initiative."@en ;
+   samm:see <https://wbcsd.github.io/tr/2023/data-exchange-protocol-20230221/> ;
+   samm:characteristic samm-c:Timestamp ;
+   samm:exampleValue "2022-05-22T21:47:32Z"^^xsd:dateTime .
+
+:status a samm:Property ;
+   samm:preferredName "Status"@en ;
+   samm:description "Status indicator of a product (carbon) footprint as specified in the technical specifications for PCF Data Exchange (Version 2.0.0) from the WBCSD (World Business Council for Sustainable Development)/ PACT initiative. WBCSD specific extension, in Catena-X set to \"Active\" per default."@en ;
```

Comment:
same here rephrase ref to catenax as example

Reply:
Added "for example" to the Catena-X reference